ts-sequelize/README.md

```diff
@@ -0,0 +1,47 @@
+# ORM Sample App
+
+## Run the app locally
+
+## Compile and Deploy with Klotho
+
+run the terminal commands:
+```sh
+npm i && npx tsc
+```
+
+```sh
+# Compile the app
+klotho . --app sample-ts-sequelize
+
+# npm install
+npm --prefix ./compiled install
+
+# Set username and password
+pulumi config set sample-ts-sequelize:typeormdb_username
```

Comment:
This will probably need a `--stack` no?

Reply:
ah i ran it from the compiled directory so it wouldnt, but yeah i probably need the -C and -s